### DISCUSSION TITLE
Open a Discord

### DISCUSSION THREAD

**Discussion by Tawagot0** [meisnate12/Plex-Meta-Manager#127] [Ideas]
I think it's a good idea to open a discord (if of course @meisnate12 uses it) to be able to communicate in real time, for example before posting an issue to be able to propose a feature request (for example I'm looking at how to integrate new options for anilist with gender) and I'd like to discuss it to better understand the code for example

**Comment by meisnate12** [comment]
I've been meaning to get around to doing this lol so https://discord.gg/NfH6mGFuAB
